plutus-core/plutus-core/src/PlutusCore/Error.hs

```diff
@@ -117,7 +119,7 @@ instance Pretty SourcePos where
 instance Pretty ParserError where
     pretty (UnknownBuiltinType s loc)       = "Unknown built-in type" <+> squotes (pretty s) <+> "at" <+> pretty loc
     pretty (BuiltinTypeNotAStar ty loc)     = "Expected a type of kind star (to later parse a constant), but got:" <+> squotes (pretty ty) <+> "at" <+> pretty loc
-    pretty (UnknownBuiltinFunction s loc)   = "Unknown built-in function" <+> squotes (pretty s) <+> "at" <+> pretty loc
+    pretty (UnknownBuiltinFunction s loc lBuiltin)   = "Unknown built-in function" <+> squotes (pretty s) <+> "at" <+> pretty loc <+> ". Parsable functions are " <+> pretty (show lBuiltin)
```

Comment:
Perhaps only pretty-print the keys of `lBuiltin`? And not via `show`, 'cause that adds quoting for `Text`.

Reply:
Yes, I agree. And if we're doing that, we might as well only store the keyset in the error.